src/Raven.Server/Documents/Indexes/Static/EsprimaVisitor.cs

```diff
@@ -562,9 +562,6 @@ public virtual void Visit(Node node)
                 case Nodes.ObjectPattern:
                     VisitObjectPattern(node.As<ObjectPattern>());
                     break;
-                case Nodes.ArrowParameterPlaceHolder:
```

Comment:
Don't we need that?

Reply:
It's internal, during parse-time special placeholder, shouldn't be used by public API.